src/issuemd-core.js

```diff
@@ -1,43 +1,43 @@
-'use strict';
+(function (root, factory) {
 
-! function () {
+    'use strict';
 
-    var root = this;
+    if (typeof define === 'function' && define.amd) {
+        // AMD. Register as an anonymous module.
+        define([], function () {
+            return (root.returnExportsGlobal = factory(root));
+        });
+    } else if (typeof exports === 'object') {
+        // Node. Does not work with strict CommonJS, but
+        // only CommonJS-like enviroments that support module.exports,
+        // like Node.
+        module.exports = factory(root);
+    } else {
+        // Browser globals
+        root.returnExportsGlobal = factory(root);
+    }
 
-    issuemd.version = '__VERSION__';
+}(typeof window !== 'undefined' ? window : this, function (root) {
 
-    issuemd.fn = require('./issuemd-methods.js')(issuemd);
+    'use strict';
 
-    function issuemd() {
+    var _issuemd = this ? this.issuemd : undefined;
 
+    var issuemd = function () {
         return issuemd.fn.main.apply(null, arguments);
+    };
 
-    }
-
-    /***************************************
-     * hook module into AMD/require etc... *
-     ***************************************/
+    issuemd.version = '__VERSION__';
 
-    if (typeof exports !== 'undefined') {
+    issuemd.fn = require('./issuemd-methods.js')(issuemd);
 
-        if (typeof module !== 'undefined' && module.exports) {
-            module.exports = issuemd;
+    issuemd.noConflict = function () {
+        if (root && root.issuemd === issuemd) {
+            root.issuemd = _issuemd;
         }
+        return this;
+    };
 
-        exports.issuemd = issuemd;
-
-    } else {
-
-        root.issuemd = issuemd;
-
-    }
-
-    if (typeof define === 'function' && define.amd) {
-
-        define('issuemd', [], function () {
-            return issuemd;
-        });
-
-    }
+    return issuemd;
 
-}();
\ No newline at end of file
+}));
```

Comment:
Actually, all looks fine - I was a bit confused because I had a few merge conflicts to deal with, so was hard to see, but now merged, looks fine.